cmd/xcaddy/main.go

```diff
@@ -76,7 +76,7 @@ func runBuild(ctx context.Context, args []string) error {
 			})
 			if repl != "" {
 				// adjust relative replacements in current working directory since our temporary module is in a different directory
-				if !filepath.IsAbs(repl) {
+				if strings.HasPrefix(repl, ".") && !filepath.IsAbs(repl) {
```

Comment:
I believe if `strings.HasPrefix(repl, ".")` is true, then `!filepath.IsAbs(repl)` is always true as well. We can probably drop the `!IsAbs()` condition.

Reply:
https://golang.org/src/path/filepath/path_unix.go?s=320:348#L2
```go
// IsAbs reports whether the path is absolute.
func IsAbs(path string) bool {
	return strings.HasPrefix(path, "/")
}
```

So yeah :+1:

Reply:
Yep, good point. Remove.